vc/pokered.patch.template

```diff
@@ -0,0 +1,482 @@
+[FPA 010 Begin]                       
+Mode = 3                              
+Type = 0                              
+Address = {HEX @}                     
+MotionBEnable0 = 3                    
+MotionBlur0 = 27                      
+ConditionType = 0                     
+ConditionValueA = {dws_ wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wAnimationID}      
+ConditionValueB = {dws_ ==            ==              ==              ==              ==          }      
+ConditionValueC = {dws_ "R"           "O"             "C"             "K"             ROCK_SLIDE  }      
+                                      
+[FPA 010 End]                         
+Mode = 3                              
+Type = 1                              
+Address = {HEX @}                    
+ConditionType = 11                     
+ConditionValueA = {dws_ wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wStringBuffer+4 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wStringBuffer+4 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wStringBuffer+5 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wAnimationID 0x00 wStringBuffer wStringBuffer+1 wStringBuffer+2 wStringBuffer+3 wAnimationID}  
+ConditionValueB = {dws_ ==            ==              ==              ==              ==           ||   ==            ==              ==              ==              ==           ||   ==            ==              ==              ==              ==           ||   ==            ==              ==              ==              ==              ==           ||   ==            ==              ==              ==              ==              ==           ||   ==            ==              ==              ==              ==              ==           ||   ==            ==              ==              ==              ==           ||   ==            ==              ==              ==              ==          }
+ConditionValueC = {dws_ "D"           "R"             "E"             "A"             DREAM_EATER  0x00 "R"           "O"             "C"             "K"             ROCK_SLIDE   0x00 "S"           "P"             "O"             "R"             SPORE        0x00 "S"           "E"             "L"             "F"             "D"             0x05         0x00 "S"           "E"             "L"             "F"             "D"             SELFDESTRUCT 0x00 "C"           "O"             "N"             "F"             "S"             CONFUSION    0x00 "E"           "X"             "P"             "L"             EXPLOSION    0x00 "E"           "X"             "P"             "L"             SUBANIM_05  }
```

Comment:
There's a hard-coded `0x05` which should also be `MEGA_PUNCH`.

Reply:
What if instead of `0x00` above+below `||` (which looks like the unidentified raw hex values), it were `00` to line up with the `||`s? Could also put two spaces around that column instead of one.